helpers/session/session.go

```diff
@@ -18,3 +18,18 @@ func Flashes(c *gin.Context) []interface{} {
 
 	return flashes
 }
+
+func Get(c *gin.Context, key string) interface{} {
+	session := sessions.Default(c)
+	return session.Get(key)
+}
+
+func Set(c *gin.Context, key string, value interface{}) {
+	session := sessions.Default(c)
+	session.Options(sessions.Options{
+		MaxAge: 3600 * 24, // 24 hours
```

Comment:
minor, but I'm not a big fan of this hardcoded value way 🙇  perhaps you can declare a const?

`const sessionTimeoutInSec = 24 * 60 * 60`

Reply:
Fixed in d1b2b811f023220b13f5fe50ac93bac97c996c80